rebar.config

```diff
@@ -13,7 +15,7 @@
   {basho_stats, ".*", {git, "git://github.com/basho/basho_stats.git", {tag, "1.0.3"}}},
   {riak_sysmon, ".*", {git, "git://github.com/basho/riak_sysmon.git", {branch, "develop"}}},
   {riak_ensemble, ".*", {git, "git://github.com/basho/riak_ensemble", {branch, "develop"}}},
-  {pbkdf2, ".*", {git, "git://github.com/basho/erlang-pbkdf2.git", {tag, "2.0.0"}}},
+  {pbkdf2, ".*", {git, "git://github.com/basho/erlang-pbkdf2.git", {tag, "develop"}}},
```

Comment:
Chris, there is no such "develop" branch for basho/erlang-pbkdf2 right now.  I can get it to compile with "master", but I dunno if that's the right branch to use there, sorry.